test/test_torch.py

```diff
@@ -16575,6 +16577,12 @@ def test_exp_slow(self, device, dtype):
         b = torch.exp(torch.ones(1, dtype=dtype, device=device))
         self.assertEqual(a, b.expand(2 ** 31))
 
+    @dtypes(*(torch.testing.get_all_int_dtypes() + torch.testing.get_all_complex_dtypes()))
```

Comment:
You don't need to add this test. By creating an OpInfo for exp2 its supported dtypes are automatically tested.

Reply:
Done.